packages/wonder-blocks-form/src/components/choice-internal.tsx

```diff
@@ -12,7 +12,7 @@ import RadioCore from "./radio-core";
 
 type Props = AriaProps & {
     /** Whether this choice is checked. */
-    checked: boolean;
+    checked: boolean | null | undefined;
```

Comment:
**suggestion:** By moving `Checked` to `types.ts`, you can do `import type {Checked} from "./util/type"`, then:

```suggestion
    checked: Checked;
```